blink/src/main/java/com/example/androidthings/simplepio/BlinkActivity.java

```diff
@@ -84,8 +85,14 @@ public void run() {
             }
             try {
                 // Toggle the GPIO state
-                mLedGpio.setValue(!mLedGpio.getValue());
-                Log.d(TAG, "State set to " + mLedGpio.getValue());
+                if (!BoardDefaults.isNxpBoard()) {
```

Comment:
I think we could make that change for all boards, and get rid of this extra check? What do other thinks?